File: api-microservice/src/main/java/io/github/brunogabriel/apimicroservice/domain/Service.java
package io.github.brunogabriel.apimicroservice.domain;

import lombok.Data;

import java.util.List;

/**
 * Created by brunogabriel on 2019-06-18.
 */
@Data
public class Service {
    public Long id;
    public Long petId;
    public String type;
    public String duration;
    public String date;
    public double price;
    public String doneBy;
    public List<String> products;
    public List<String> annotations;
}

File: DoggieApp/app/src/main/java/io/github/brunogabriel/doggieapp/shared/models/Service.kt
package io.github.brunogabriel.doggieapp.shared.models

/**
 * Created by brunogabriel on 2019-06-19.
 */
data class Service(
    val id: Long,
    val petId: Long,
    val type: String,
    val duration: Int,
    val price: Double,
    val date: String,
    val employee: String,
    val products: List<String>?,
    val annotations: List<String>?)